infra/growth/src/scripts/banParticipants.js

```diff
@@ -17,27 +17,32 @@ const { SdnMatcher } = require('../util/sdnMatcher')
 Logger.setLogLevel(process.env.LOG_LEVEL || 'INFO')
 const logger = Logger.create('banParticipants', { showTimestamp: false })
 
-let FraudEngine, employeesFilename
+let FraudEngine, employeesFilename, trustedFilename
 if (process.env.NODE_ENV === 'production' || process.env.USE_PROD_FRAUD) {
   FraudEngine = require('../fraud/prod/engine')
   employeesFilename = `${__dirname}/../fraud/prod/data/employees.txt`
+  trustedFilename = `${__dirname}/../fraud/prod/data/trusted.txt`
   logger.info('Loaded PROD fraud engine.')
 } else {
   FraudEngine = require('../fraud/dev/engine')
   employeesFilename = `${__dirname}/../fraud/dev/data/employees.txt`
+  trustedFilename = `${__dirname}/../fraud/dev/data/trusted.txt`
   logger.info('Loaded DEV fraud engine.')
 }
 
 /**
  * Helper class that loads the list of Origin employees accounts.
- * These acounts are exempt from fraud check but also do not get any reward payout.
+ * These accounts are exempt from fraud check but also do not get any reward payout.
  */
 class OriginEmployees {
   constructor(filename) {
     this.addresses = {}
     const data = fs.readFileSync(filename).toString()
     const lines = data.split('\n')
     for (const line of lines) {
+      if (line.startsWith('#')) {
```

Comment:
nit: can be replaced with: `line.match(/\s+#.+/g)` so that hash tag can have preceding spaces and also be ignored

Reply:
Yes. good idea. thank you. done.